lightning-block-sync/src/init.rs

```diff
@@ -61,16 +61,16 @@ BlockSourceResult<ValidatedBlockHeader> where B::Target: BlockSource {
 ///
 /// async fn init_sync<
 /// 	B: BlockSource,
-/// 	K: KeysInterface<Signer = S>,
+/// 	K: KeysInterface,
 /// 	S: keysinterface::Sign,
```

Comment:
Looks like this can be dropped

Reply:
that's so odd. This was literally part of my search string; I'm now paranoid that I missed a bunch of other things.

Reply:
Maybe the doc comments threw it off.